### PR TITLE
Delete pip install support script & use a manual tag for update docke…

### PR DESCRIPTION
**Pull request by d1jang**
…rfile.

We have decided to support pip install by converting the current read-only filesystem into a writable one.

This PR introduces a tag `pinned` that any emergency dockerfile like this one should be based on. If an updating dockerfile depended on `latest`, it will keep adding new changes on top of its previous added changes, which may result in unwanted problems.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -521,14 +521,4 @@ ENV PYTHONUSERBASE "/root/.local"
...
-# Set up an initialization script to be executed before any other commands initiated by the user.
-ADD patches/entrypoint.sh /root/entrypoint.sh
-RUN chmod +x /root/entrypoint.sh
-# NOTE: ENTRYPOINT set by "FROM <image>" should preceed the our own custom entrypoint.
-# Currently tini is set as ENTRYPOINT for the base image, and it can be combined with our entrypoint (https://github.com/krallin/tini).
-# ENTRYPOINT gets executed by "docker run <image> <command>" and it runs <command> at the end of its execution.
-# Make sure tini exists.
-RUN /usr/bin/tini -h > /dev/null
-ENTRYPOINT ["/usr/bin/tini", "--", "/root/entrypoint.sh"]
+ENV MPLBACKEND "agg"
```
Comment (rosbo) [inline Dockerfile]: add newline

File: kaggle_tools_update.Dockerfile [hunk 2]
```diff
@@ -9,23 +9,12 @@
 # is enabled.
 
 # Usage:
-#   docker rmi gcr.io/kaggle-images/python:latest
+#   docker rmi gcr.io/kaggle-images/python:pinned
```
Comment (rosbo) [inline kaggle_tools_update.Dockerfile]: You should add instructions on how to pin the current stable image

File: kaggle_tools_update.Dockerfile [hunk 3]
```diff
@@ -9,23 +9,12 @@
...
-ADD patches/entrypoint.sh /root/entrypoint.sh
-RUN chmod +x /root/entrypoint.sh
-# NOTE: ENTRYPOINT set by "FROM <image>" should preceed the our own custom entrypoint.
-# Currently tini is set as ENTRYPOINT for the base image, and it can be combined with our entrypoint (https://github.com/krallin/tini).
-# ENTRYPOINT gets executed by "docker run <image> <command>" and it runs <command> at the end of its execution.
-# Make sure tini exists.
-RUN /usr/bin/tini -h > /dev/null
-ENTRYPOINT ["/usr/bin/tini", "--", "/root/entrypoint.sh"]
\ No newline at end of file
+RUN pip install --upgrade git+https://github.com/Kaggle/learntools
```
Comment (rosbo) [inline kaggle_tools_update.Dockerfile]: newline

File: Dockerfile (hunk 1 above)
Comment (d1jang) [reply Dockerfile]: Done.

File: kaggle_tools_update.Dockerfile (hunk 2 above)
Comment (d1jang) [reply kaggle_tools_update.Dockerfile]: Done.

File: kaggle_tools_update.Dockerfile (hunk 3 above)
Comment (d1jang) [reply kaggle_tools_update.Dockerfile]: Done.

**Pull request merged**